proof/refine/VSpace_R.thy

```diff
@@ -7,8 +8,206 @@
 
 (* VSpace refinement - generic interface *)
 
+(* FIXME arch-split: arguably most of this can be moved elsewhere and this file deleted, making
+   VSpace_R an arch-only theory *)
```

Comment:
was hoping to get some opinions on this one; @lsf37 any thoughts to add?

Reply:
Eventually that could happen, yes, but I wouldn't see it as a priority right now. There's also a chance that we'll discover other things during the arch split run that should maybe go here, so I'd leave this FIXME for the end.